### DISCUSSION TITLE
Lazyvim crash unexpected

### DISCUSSION THREAD

**Discussion by huyphan-mesoneer** [LazyVim/LazyVim#4114] [Q&A] [answered]
Recently, my lazyvim suddenly crash but I do not know the reason. Does anyone know where can I check logs of anything that know why it crash?

But it's quite strange because it just crash randomly when I edit terraform project. I do not meet this issue while working with Angular or TS projects. I do not change any configs for several months.

OS: MacOS
Issue with language: HCL

**Comment by folke** [comment, marked as answer]
Check the Neovim wiki on how to create a debug build and get a stack trace.
Segfaults won't be logged, since your program crashed. 
Segfaults are also always due to bugs in Neovim, so not in LazyVim.

**Reply by huyphan-mesoneer** [reply]
@folke ok thanks
